src/components/App.tsx

```diff
@@ -4,23 +4,32 @@ import CssBaseline from '@mui/material/CssBaseline';
 import { ThemeProvider } from '@mui/material/styles';
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 
+import store from '../common/store';
 import protocolTheme from '../theme/protocolTheme';
+import { fetchRecentRevisions } from '../thunks/searchThunk';
```

Comment:
You can now add as param` hide_reviewbot_pushes=true` to endpoint call to exclude results for reviewbot